site/src/Librecores/ProjectRepoBundle/Validator/Constraints/UserOrgNameValidator.php

```diff
@@ -76,7 +76,7 @@ class UserOrgNameValidator extends ConstraintValidator
      *
      * @var bool
      */
-    const CHECK_UNIQUE_ORG  = true;
+    const CHECK_UNIQUE_ORG  = false;
```

Comment:
I knew you were going to FOSDEM and so I accelerated development (we have a great 'marketing' opportunity at FOSDEM!). I agree we should finish this PR sooner or later and Bernd hopefully will find the time to beautify it before next weekend.